backend/api/Areas/Acquisition/Controllers/AcquisitionFileController.cs

```diff
@@ -53,6 +59,16 @@ public AcquisitionFileController(IAcquisitionFileService acquisitionService, IMa
         [SwaggerOperation(Tags = new[] { "acquisitionfile" })]
         public IActionResult GetAcquisitionFile(long id)
         {
+            // RECOMMENDED - Add valuable metadata to logs
+            _logger.LogInformation("Request received by Controller: {Controller}, Action: {ControllerAction}, User: {User}, DateTime: {DateTime}",
```

Comment:
Thanks for this, do both of these seem like info level logs to you or should either of these be a debug maybe?

Reply:
As per MS recommended levels [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-6.0#log-level) I believe Information would be a better fit than Debug. Also Debug is ignored on live environments and the above statement gives us the ability to answer questions like: "I want to know which actions a particular user XYZ did on this day that resulted in an error"

![image](https://user-images.githubusercontent.com/24322224/186223345-46de4cb7-b32c-43b7-b2e1-add7f17e63ab.png)